device/fw_debug_device.go

```diff
@@ -0,0 +1,45 @@
+package device
+
+import (
+	"fmt"
+)
+
+const (
+	FWDebugPort = uint64(0x402)
+)
+
+type FWDebugDevice struct {
+	Port uint64
+}
+
+func (f *FWDebugDevice) Read(port uint64, data []byte) error {
+	if len(data) == 1 {
+		data[0] = 0xE9
+	} else {
+		return errDataLenInvalid
+	}
+
+	return nil
+}
+
+func (f *FWDebugDevice) Write(port uint64, data []byte) error {
+	if len(data) == 1 {
+		if data[0] == '\000' {
+			fmt.Printf("\r\n")
+		} else {
+			fmt.Printf("%s", string(data[0]))
```

Comment:
I think this can be replaced with the code below:
```suggestion
			fmt.Printf("%c", data[0])
```

Reply:
done